### PR TITLE
Fix comma placement in confirmation screen

### LINKED ISSUE #159: Practice address displays punctuated improperly
During provider enrollment, on the Step 4 ("Summary") screen, in the "practice information" confirmation display, the final line of the practice address displays as (example):

`Minneapolis ,MN ,39582 ,40`

This has an extra number after the ZIP code (perhaps the county code?) and has spaces before commas, instead of spaces after commas. Also, there shouldn't be a comma between the state abbreviation and the ZIP code.

practice-information-commas

### PR DESCRIPTION
**Pull request by brainwane**
During provider enrollment, on the Step 4 ("Summary") screen, in the
"practice information" confirmation display, the final line of the
practice address should have a comma and then a space between the city
name and the state, and no comma between the state and the ZIP code.

The old behavior would omit a comma if the state field were empty.
Jason and I decided that, in the unlikely event that a user somehow
got to the end of the process without specifying a state, then having
an address with a trailing comma after the city was a reasonable
trade-off.

Fixes #159.

### CODE REVIEW

**Comment by brainwane** [comment]
Screenshots of the new behavior:

group-practice-commas
proper-commas-private

File: psm-app/cms-web/WebContent/WEB-INF/pages/provider/enrollment/steps/pageTemplates/common/primary_practice.jsp [hunk 1]
```diff
@@ -21,9 +21,9 @@
     <span>
         <c:if test="${not empty requestScope['_06_addressLine1']}"><c:out value="${requestScope['_06_addressLine1']}" /><br /></c:if>
         <c:out value="${requestScope['_06_addressLine2']}" /><br />
-        <c:set var="city" value="${requestScope['_06_city']}" /><c:out value="${city}" /> 
-        <c:set var="state" value="${requestScope['_06_state']}" /><c:if test="${not empty state}">,</c:if>${state}
-        <c:set var="zip" value="${requestScope['_06_zip']}" /><c:if test="${not empty zip}">,</c:if>${zip} 
+        <c:set var="city" value="${requestScope['_06_city']}" /><c:out value="${city}" />,
+        <c:set var="state" value="${requestScope['_06_state']}" /><c:if test="${not empty state}"> </c:if>${state}
+        <c:set var="zip" value="${requestScope['_06_zip']}" /><c:if test="${not empty zip}"> </c:if>${zip}
```
Comment (jasonaowen) [inline psm-app/cms-web/WebContent/WEB-INF/pages/provider/enrollment/steps/pageTemplates/common/primary_practice.jsp]: Do we still need the `<c:if test="${not empty ...}"> </c:if>`? Now that we're always displaying the commas, and HTML treats all whitespace the same, and there's a newline and a bunch of spaces after the comma, I think we can delete the extra optional space.

That, in turn, I think means we don't need to `c:set` a variable, since we're only using it once - we can just inline it to its use.

tl;dr: can we replace these lines with something like this?
```
${requestScope['_06_city']}, ${requestScope['_06_state']} ${requestScope['_06_zip']}
```

Comment (brainwane) [inline psm-app/cms-web/WebContent/WEB-INF/pages/provider/enrollment/steps/pageTemplates/common/primary_practice.jsp]: Sounds great!

**Comment by brainwane** [comment]
Updated to take @jasonaowen's suggestion, and to check that the comma fix is also in place for the billing/reimbursement addresses.
